docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -1334,6 +1334,19 @@ Query Parameters:
 
 -   **Content-type** – Set to `"application/tar"`.
 -   **X-Registry-Config** – base64-encoded ConfigFile object
+-   **X-Docker-BuildArgs** – base64-encoded JSON map of string pairs for build-time
```

Comment:
my thought was to not add *more*; using headers to send data is (IMO), well, weird? Also, we're already using query params.